crates/turborepo-lockfiles/Cargo.toml

```diff
@@ -14,7 +14,7 @@ regex = "1"
 semver = "1.0.17"
 serde = { version = "1.0.126", features = ["derive", "rc"] }
 serde_json = "1.0.86"
-serde_yaml = "0.9"
+serde_yaml = "0.9.27"
```

Comment:
Pinning this to an exact `serde_yaml` version since we are now aware that we depend on `x: String` handles input like `x: 2` which can get affected by `serde` container attributes. I'm not sure which behavior is intended/buggy so we might be depending on broken behavior. (Unit tests would catch if an upgrade broke us, but just in case let's pin it)